docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,293 @@
+# Storage with Rook Ceph on Packet cloud
+
+## Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Deploy storage worker pool](#deploy-storage-worker-pool)
+  - [Config](#config)
+  - [Deploy the worker pool](#deploy-the-worker-pool)
+- [Deploy Rook](#deploy-rook)
+  - [Config](#config-1)
+  - [Deploy the component](#deploy-the-component)
+- [Deploy Rook Ceph](#deploy-rook-ceph)
+  - [Config](#config-2)
+  - [Deploy the component](#deploy-the-component-1)
+- [Access the Ceph dashboard](#access-ceph-dashboard)
+- [Enable and access toolbox](#enable-and-access-toolbox)
+- [Enable monitoring](#enable-monitoring)
+- [Make default storage class](#make-default-storage-class)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides the steps for deploying a storage stack using the `rook` and `rook-ceph` Lokomotive component and explains how to access Ceph dashboard, Ceph toolbox and how to enable monitoring.
+
+## Prerequisites
+
+* A Lokomotive cluster deployed on a supported provider and accessible via `kubectl`.
+
+## Deploy storage worker pool
+
+#### Config
+
+Deploy a cluster with at least one worker pool dedicated to `rook-ceph`. A dedicated worker pool configuration should look like the following:
+
+```tf
+cluster "packet" {
+  ...
+
+  worker_pool "storage" {
+    count = 3
+    node_type = "c2.medium.x86"
+
+    labels = "storage.lokomotive.io=ceph"
+    taints = "storage.lokomotive.io=ceph:NoSchedule"
+  }
+}
+```
+
+- The number of machines provided using `count` should be an odd number greater than three.
```

Comment:
The example has `count = 3`, which does not satisfy `greater than three`. Should we change one or another?

Reply:
yep it needs an addendum of `or equal to`.